app/api/business/brief/brief_edit_business.py

```diff
@@ -47,7 +47,8 @@ def get_opportunity_to_edit(user_id, brief_id):
     return {
         'brief': brief.serialize(with_users=False),
         'domains': domains,
-        'isOpenToAll': brief_business.is_open_to_all(brief)
+        'isOpenToAll': brief_business.is_open_to_all(brief),
+        'blackout_period': brief_blackout_validator.get_blackout_dates(formatted=True)
```

Comment:
I think this needs to be updated to call from `brief_business`.

Reply:
Yes, this needs to be fixed.